requirements.txt

```diff
@@ -1,2 +1,2 @@
 # To edit the requirements file, open a PR on github.com/DataDog/datadog-agent-buildimages
--r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements.txt
+-r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/jaime/golang.1.19.9/requirements.txt
```

Comment:
This has to be reverted once https://github.com/DataDog/datadog-agent-buildimages/pull/382 is merged.